src/metabase/driver/sql_jdbc/actions.clj

```diff
@@ -496,7 +524,8 @@
 (mr/def ::table-row-input
   [:map
    [:table-id ::lib.schema.id/table]
-   [:row ::lib.schema.actions/row]])
+   [:row ::lib.schema.actions/row]
+   [:delete-children? {:optional true} [:maybe :boolean]]])
```

Comment:
I know we're already in this territory with hyphens, but having question marks in input keys feels a bit JS / HTTP hostile. Just thinking out loud - in practice this is totally fine, as the we're going to end up building the forms on the server side and the FE won't have to hard code the key, and it will never be an URL parameter.

Let's keep it like this for now. Maybe at a later stage of the project we will consider more "API centric" signatures for all our actions and can standardize in one sweep.